src/content/docs/tutorial-reporting-data/capture-the-right-data.mdx

```diff
@@ -0,0 +1,121 @@
+
+Revealing improperly instrumented services to users creates a situation that can put customer satisfaction at risk. This burn-in period is difficult to escape as new features can come from code bases without a strong link between software delivery and observability programs. Service instrumentation planning is the approach used to describe a single service runtime through telemetry, and this guide focuses on the metrics of your application's code as well as external measurements through synthetic testing. 
+
+You're a good candidate for using this guide if any of the following are true:
+* Your development teams are disconnected from production observability design.
+* You have new services/capabilities that run in production and before fully establishing telemetry and alerting.
+* You need to provide additional business context to your instrumentation to improve diagnosis and business KPI measurement.
+* You employ a highly customized or proprietary software framework.
+* Your service is under active development. Legacy services, and services built from commercial-off-the-shelf platforms, tend to be better served with generic instrumentation options.
+
+## Understand the benefits [#understand-benefits]
+
+Involving developers in this process has the following benefits:
+
+1. Improved troubleshooting:
+  * Good telemetry naming gives operations staff a common language to use with developers during incidents, reducing the time to triage and fix problems.
+  * More precise and contextually relevant telemetry from your service allows for more accurate detection of faults that you can take action on.
+
+2. Better informed development decisions by:
+  * Detecting areas of volatility or unexpected behavior and addressing them.
+  * Understanding what dependencies in your code lack redundancy and taking measures to improve the service.
+  * Appreciating how end-users are employing your software. You can better understand where improvements will have the biggest impact.
+
+## Use Key performance indicators [#key-perf-indicators]
+
+It's important know a few simple KPIs to track the ongoing improvements in your software delivery and operations programs. Here are two main types of KPIs to consider as you improve instrumentation:
+
+* **Business KPIs** are aligned to your overall program objectives and should be consistently measured to demonstrate ongoing improvement for each service. Business KPIs include: 
+
+<CollapserGroup>
+  <Collapser
+    id="kpi-service-quality"
+    title="Service quality"
+  >
+A metric is required to define how well your service is operating. This will depend upon the needs of your organization and the constraints of the services being monitored.
```

Comment:
This tutorial is a bit over my head to be honest so take all this feedback with a grain of salt -

Should this be "A metric required..." or "This is a metric required"?

Reply:
Looks good to me, I actually tweaked this a bit as well to remove any obfuscation that comes from the wording